package.json

```diff
@@ -273,12 +286,25 @@
     "mocha": "^5.2.0",
     "phantomjs-polyfill": "^0.0.2",
     "phantomjs-prebuilt": "^2.1.16",
+    "postcss-simple-vars": "^5.0.2",
     "request": "^2.88.0",
+    "rollup": "^1.6.0",
```

Comment:
we should kill this right?

Reply:
yes, I'll cleanup rollup config & dependencies. let's stick to webpack for now.